contracts/other/yieldspace-tv/YVTokenMultiConverter.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.14;
+
+import "./IConverter.sol";
+import "../../oracles/yearn/IYvToken.sol";
+import "@yield-protocol/utils-v2/contracts/token/IERC20.sol";
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+
+/// @title A contract to convert between yvToken to underlying & vice versa
+contract YVTokenMultiConverter is IConverter, AccessControl {
+    /// @notice Mapping of yvToken to underlying asset
+    mapping(IYvToken => IERC20) yvTokenMapping;
```

Comment:
```suggestion
    mapping(IYvToken => IERC20) wrappers;
```

Not in love with `wrappers`, but convention is to use a plural, and if we are making this a pattern, we want to make it general.